vllm/assets/monitors/token_throughput.json

```
@@ -0,0 +1,33 @@
+{
+    "version": 2,
+    "created_at": "2024-06-26",
+    "last_updated_at": "2024-06-26",
+    "title": "vLLM application token usage is high",
+    "description": "The more tokens a language model generates per second the more much work it is doing. This monitor will alert you if a model that vLLM is serving is generating a lot of tokens, this indicates high load.",
```

Thanks for adding more context! If you also have impact, that would work well here too. This could be impact to costs, or slower results (I don't know what the impact of higher token usage is). This is giving users an idea for what use case they would want to use this monitor.

```suggestion
    "description": "The more tokens a language model generates per second the more work it is doing. This monitor will alert you if a model that vLLM is serving is generating a lot of tokens, this indicates high load.",
```

---

Noted! I'll do that on monday, realized I can add another monitor for this integration while I'm at it.